webapp/pages/profile/_id/_slug.vue

```diff
@@ -251,7 +251,7 @@
           </template>
         </masonry-grid>
         <div
-          v-if="hasMore"
+          v-if="hasMore && posts.length >= pageSize"
```

Comment:
Is this the bug? Was the `hasMore` not enough? I can see that this condition will be `false` initially. But how relates that to the bug?